pkg/scheduler/event_handler.go

```diff
@@ -210,6 +210,8 @@ func (s *Scheduler) updateCluster(oldObj, newObj interface{}) {
 		// to the worker. Therefore, call Add func instead of Enqueue func.
 		s.clusterReconcileWorker.Add(oldCluster)
 		s.clusterReconcileWorker.Add(newCluster)
+	case !equality.Semantic.DeepEqual(oldCluster.Status.ResourceSummary, newCluster.Status.ResourceSummary):
+		s.clusterResourcesReconcileWorker.Add(newCluster)
```

Comment:
How about calling `Enqueue` here?

Reply:
It may be impossible to use `Add` here to add the entire cluster object for subsequent use.

Reply:
ok.